gradle.properties

```diff
@@ -15,7 +15,7 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 # Android operating system, and which are packaged with your app"s APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
-
+android.enableJetifier=true
```

Comment:
Did you rebase or merge with previous changes? This one was removed and I don't think should be added again

Reply:
this was added because at the time, the auth sdk supporting androidx was not available. Once the new version of the native sdk is out, it can be removed

Reply:
Soon!!! 😁